docker_config/install.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# install.sh used for docker image build
```

Comment:
This script seems the same as the existing `install.sh`? If so, can we remove it?

Reply:
Yes I think we can remove it. I changed the environment variable in it because the path for FedScale is fixed in images. But a better way to do is to just define a `$FEDSCALE_HOME` in our Dockerfiles.